src/components/Layout/TestLayout.tsx

```diff
@@ -0,0 +1,64 @@
+import { PropsWithChildren, useState } from "react";
+import { IconButton } from "src/components/IconButton";
+import { NestedScrollLayoutContextProvider } from "src/components/Layout/NestedScrollLayoutContext";
+import { PreventScrollContainer } from "src/components/Layout/PreventScrollContainer";
+import { ScrollableContent } from "src/components/Layout/ScrollableContent";
+import { Css, Margin, Only, Padding, Xss } from "src/Css";
+import { zeroTo } from "src/utils/sb";
+
+export function TestLayout({ children }: PropsWithChildren<any>) {
+  return (
+    <PreventScrollContainer>
+      <TestTopNav />
+      <main css={Css.df.mh0.$}>{children}</main>
```

Comment:
Both `df` and `mh0` are required here to ensure the main content shows the scrollbar and doesn't exceed the remaining height of the viewport.

Reply:
Since this is necessary to do along side of `<PreventScrollContainer />`, maybe  this `<main />`(or optionally a different element) with these styles should be included via something like `<PreventScrollContainerContent />`? Hmm.. not sure about this. Would be nice if all apps didn't need to know which styles will allow the scrolling within `main`, but can't think of any good naming.